tests/transport/SubjectInboundTransport.ts

```diff
@@ -48,7 +48,16 @@ export class SubjectInboundTransport implements InboundTransport {
           })
         }
 
-        await messageReceiver.receiveMessage(message, { session })
+        // This emits a custom error in order to easily catch in E2E tests when a message
+        // reception throws an error. TODO: Remove as soon as agent throws errors automatically
```

Comment:
Yeah it would be good if we can extend the current event to also emit when the processing went wrong. There's an issue open for it